Fetch from:
  https://git.kernel.org/cgit/utils/kernel/kexec/kexec-tools.git/patch/?id=9bcefc97bc6c03b6acc8c25f6b1d4e796521ea4c

From 9bcefc97bc6c03b6acc8c25f6b1d4e796521ea4c Mon Sep 17 00:00:00 2001
From: Tony Jones <tonyj@suse.de>
Date: Wed, 05 Feb 2014 22:32:18 +0000
Subject: i386: fix build failure (bzImage_support_efi_boot)

Commit 9c200a85de2245a850546fded96a1977b84ad24d referenced
'bzImage_support_efi_boot' without matching 32-bit definition.

Signed-off-by: Tony Jones <tonyj@suse.de>
Signed-off-by: Simon Horman <horms@verge.net.au>
---
diff --git a/kexec/arch/i386/kexec-bzImage.c b/kexec/arch/i386/kexec-bzImage.c
index fc1a54f..e7bc1d6 100644
--- a/kexec/arch/i386/kexec-bzImage.c
+++ b/kexec/arch/i386/kexec-bzImage.c
@@ -40,6 +40,7 @@
 #include <arch/options.h>

 static const int probe_debug = 0;
+int bzImage_support_efi_boot = 0;

 int bzImage_probe(const char *buf, off_t len)
 {
--
cgit v0.9.2
